modules/nw-external-dns-operator.adoc

```diff
@@ -50,3 +50,43 @@ $ oc get -n external-dns-operator deployment/external-dns-operator
 NAME                    READY     UP-TO-DATE   AVAILABLE   AGE
 external-dns-operator   1/1       1            1           23h
 ----
+
+== External DNS Operator domain name limitations
```

Comment:
This is not a strong opinion but maybe this chapter can be moved to the bottom, after the operator logs' chapter.

Reply:
Addressed